### DISCUSSION TITLE
Storytime Animation Done In ToonSquid

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#451] [Show and tell]
[Rode Mic vs. iPad Mic (My First Storytime Animation)](https://youtu.be/rO2bG9HGenI?si=OsfU2OSD3F5-oiRv) I've been wanting to do my own Storytime animation for a long time now. So, with the help of ToonSquid, and several work arounds, I managed to create a Storytime animation on iPad. I'd love to hear what you think.

**Comment by kevincreative** [comment]
Rode mic was better. I liked your character too.
